src/java/eu/rssw/pct/oedoc/OpenEdgeDocumentation115.java

```diff
@@ -118,7 +120,7 @@ public void execute() {
 
         // Destination directory must exist
         if (this.destDir == null) {
-            throw new BuildException(Messages.getString("OpenEdgeClassDocumentation.0"));
+            throw new BuildException(MessageFormat.format(Messages.getString("OpenEdgeClassDocumentation.0"), "destDir"));
```

Comment:
I currently took a copy of the OpenEdgeDocumentation115.java class to write ABLDuck.java, i could look at inheriting it instead :+1: